src/QirTools/pyqir/qirlib/src/emit/qir/mod.rs

```diff
@@ -0,0 +1,35 @@
+use inkwell::memory_buffer::MemoryBuffer;
+use inkwell::module::Module;
+use inkwell::values::FunctionValue;
+
+use super::Context;
+
+pub mod array1d;
+pub mod basic_values;
+pub mod calls;
+pub mod instructions;
+pub mod qubits;
+
+pub(crate) fn get_entry_function<'ctx>(context: &Context<'ctx>) -> FunctionValue<'ctx> {
+    let ns = "QuantumApplication";
```

Comment:
Is this namespace name fixed, then? Would it make sense to allow that to be user-configurable?